src/lib/mina_metrics/prometheus_metrics/mina_metrics.ml

```diff
@@ -1559,7 +1559,7 @@ module Object_lifetime_statistics = struct
     Gauge_map.add lifetime_quartile_ms_table ~name ~help
 end
 
-module Execution_times = struct
+(* module Execution_times = struct
```

Comment:
This module seemingly performs side effects, so it probably shouldn't be removed.